core/services/ocr2/plugins/ocr2keeper/evm21/abi.go

```diff
@@ -26,10 +26,6 @@ const (
 	UPKEEP_FAILURE_REASON_MERCURY_CALLBACK_REVERTED
 	UPKEEP_FAILURE_REASON_REVERT_DATA_EXCEEDS_LIMIT
 	UPKEEP_FAILURE_REASON_REGISTRY_PAUSED
-
-	// Start of offchain failure types. All onchain failure reasons from
-	// contract should be put above
-	UPKEEP_FAILURE_REASON_MERCURY_ACCESS_NOT_ALLOWED
```

Comment:
offchain failure reason is no longer used

Reply:
@FelixFan1992 also true for mercury v0.2?

Reply:
plugin does not look at the failure reason anymore.
the only use case here is if failure reason is target check reverted, we will attempt feed lookup. otherwise, we skip this step.